packages/wonder-blocks-core/components/clickable.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {Link} from "react-router-dom";
```

Comment:
maybe arriving a bit late to the party.... just wondering, is there a strong reason for including `react-router-dom` as a WB dependency? instead of letting `webapp` handling directly the routing? Asking these questions b/c I would like to understand the context 😃

Reply:
I wish we didn't, but we wanted to be able to use client-side routing with wonder-blocks components and being able to set `href="/foo/bar"` on components seemed a convenient way to specify routes.  I couldn't figure how to make this work without importing `Link` from `react-router-dom` within wonder-blocks. 😅